coderd/database/migrations/000296_notifications_inbox.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE inbox_notifications (
+	id 				UUID 						PRIMARY KEY,
+	user_id 		UUID 						NOT NULL REFERENCES users(id) ON DELETE CASCADE,
+	template_id 	UUID 						NOT NULL REFERENCES notification_templates(id) ON DELETE CASCADE,
+	targets 		UUID[],
+	title			TEXT 						NOT NULL,
+	content 		TEXT 						NOT NULL,
+	icon 			TEXT 						NOT NULL,
+	actions 		JSONB 						NOT NULL,
```

Comment:
Given that you have `title` and `content`, I assume you're going to be accepting the fully rendered content (probably Markdown) of the notification in these fields. That means that CTAs will already be included in `content`, I think. What role will `actions` play here?

Reply:
The idea was to generate the title, content and actions using a `Dispatcher` method [similar to SMTP](https://github.com/coder/coder/blob/main/coderd/notifications/dispatch/smtp.go#L63).
the `payload` field contains everything to have these three elements.

The reason why I had in mind to keep the actions separated from the body is that , based on the client (VSCode for example) we will not be able to handle the actions the same way. If we integrate the actions in the body using markdown it will not be compatible with everything. wdyt ?